LQResultProducer.py

```diff
@@ -6291,7 +6291,8 @@ def MakeBasicPlot(recovariable,xlabel,presentationbinning,selection,weight,FileD
 		leg.AddEntry(hs_rec_WJets,'W + jets')
 	leg.AddEntry(hs_rec_TTBar,'t#bar{t}' + (' (e #mu est)')*('TTBarDataDrivena' in tagname))
 
-	leg.AddEntry(hs_rec_DiBoson,'Other background')
+	leg.AddEntry(hs_rec_DiBoson,'VV and t#bar{t}V')
```

Comment:
I would change to VV+ttV

Reply:
Sure thing. On that note, should we also use "W+jets + Single Top" instead of "Other background?"

Reply:
I was debating. At final selection there is no W+jets right? But at preselection there is......

At final selection it could be just be single top, or t+jets

Reply:
Yeah, there's no W+Jets at final selection. I can change to single top or t+jets for the final sel. plots for now, and you can let me know what to do for the preselection plots

Reply:
Ok sure, let's make it W+jets + Single Top at preselection, Single Top at final selection